qokit/sk.py

```diff
@@ -0,0 +1,64 @@
+###############################################################################
+# // SPDX-License-Identifier: Apache-2.0
+# // Copyright : JP Morgan Chase & Co
+###############################################################################
+"""
+Helper functions for the S_k problem
+"""
+from qokit.fur.qaoa_simulator_base import TermsType
+import numpy as np
+import networkx as nx
+
+
+def sk_obj(x: np.ndarray, J: np.ndarray) -> float:
+    """Compute the value of a cut.
+    Args:
+        x (numpy.ndarray): binary string as numpy array.
+        w (numpy.ndarray): adjacency matrix returned by get_adjacency_matrix
```

Comment:
Not an adjacency matrix, not `w`. Should be something like "matrix specifying the couplings in the SK model"

Reply:
Done!